src/contrib/cluster/Akka.DistributedData/DistributedData.cs

```diff
@@ -83,10 +85,23 @@ public DistributedData(ExtendedActorSystem system)
             else
             {
                 var name = config.GetString("name", null);
-                Replicator = system.ActorOf(Akka.DistributedData.Replicator.Props(_settings), name);
+                Replicator = system.ActorOf(GetSupervisedReplicator(_settings, name), name+"Supervisor");
             }
         }
 
+        private static Props GetSupervisedReplicator(ReplicatorSettings settings, string name) => BackoffSupervisor.Props(
```

Comment:
The impact this has on the actor path should not affect replication - HOWEVER, when introducing Akka.NET v1.4.22 into a cluster that is running v1.4.21, the replication system will break temporarily due to:

https://github.com/akkadotnet/akka.net/blob/dev/src/contrib/cluster/Akka.DistributedData/Replicator.cs#L1104

Reply:
We should introduce a HOCON config setting:

```
akka.cluster.distributed-data.recreate-on-failure = off
```

When that setting is off, we don't use the `BackoffSupervisor` - and we will keep that setting to `off` by default. That way this change can be introduced in a non-breaking fashion. When you want to use the `BackoffSupervisor`, set this setting to `on`.
```

Reply:
I've done that change, to default to the original implementation, with a setting to enable backoff.